src/commands/general/say.rs

```diff
@@ -21,9 +22,17 @@ pub async fn say(
 		.await
 		.ok_or_eyre("Couldn't get channel!")?;
 
+	if let Context::Prefix(prefix) = ctx {
+		// ignore error, we might not have perm
```

Comment:
required_permissions makes sure we have permissions, no?

Reply:
for the bot to delete messages?

Reply:
brain fart, thought this was checking if we should actually run the command

anyways though, i just realized this is basically pointless and recreating ephemeral slash commands. it'd be better to just use that built in behavior, especially since we won't be deleting messages much here as everyone who can use /say is above the bot in role afaik

Reply:
You can delete messages created by somebody with a role above you

Reply:
I don't see a reason not to have this tbh
If you don't, the say command stays visible.